### PR TITLE
ENTERPRISE-400: Providing a better description and format to dates throughout docs. 

### PR DESCRIPTION
**Pull request by wootencl**
Completion of ticket [ENTERPRISE-400](https://pokitdok.atlassian.net/browse/ENTERPRISE-400) which entailed making it clearer the format of the dates in the request/response.

### CODE REVIEW

**Comment by dixonwh** [comment]
Hi Carter - thanks for making these updates. It's helped to clarify our expectations around receiving date and date information as well as what folks can expect in the response. From a documentation standpoint I think we're good to go. 

Before we merge this PR we should validate that we can submit the updated requests with a test mode application. In other words we need to send a test request using a platform test mode application for each example that's been updated. This will ensure that customers won't encounter any issues when using our documentation to send sample requests.

**Comment by wootencl** [comment]
All changed endpoint example requests were tested and work.

**Comment by wootencl** [comment]
@dixonwh Funny enough when confirming I didn't break anything with this ticket I found a few example requests that were already broken. Should be fixed with this PR: https://github.com/pokitdok/apidocs-v4/pull/107/files

File: source/includes/_tradingpartners.md [hunk 1]
```diff
@@ -144,9 +144,9 @@ The /tradingpartners/ response contains the following fields:
...
+| monitoring.{api_name}.last_updated     | {datetime}| The date the monitoring status was last updated. Field returned only for /tradingpartners/{id} request. In ISO8601 format (YYYY-MM-DDThh:mm:ssTZD).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (jeffhoek) [inline source/includes/_tradingpartners.md]: for this and all other ISO-8601 datetimes does the actual string _really_end with 'TZD'?  looking at https://github.com/pokitdok/apidocs-v4/pull/106/files#diff-6d2f27e23a884b8f77aab2912d4af356R102 this does not seem to be true.

**Pull request merged**